### PR TITLE
Updating contributing with fonts findings

### PR DESCRIPTION
**Pull request by nehasri89**

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -109,6 +109,42 @@ Follow these steps to deploy storybook native to a real android device.
...
+But the FontName will have the weight details, whatever it is. eg.
+
+TimesDigitalW04-Regular
+TimesDigitalW04-Italic
+TimesDigitalW04-Bold
+
+- Family Name format (meta of the font) : <fontname>
+The font Family name should be constant across all of the above: eg.
+
+TimesDigitalW04
```
Comment (aronshamash) [inline .github/CONTRIBUTING.md]: is the `04` needed? Can we remove for a nicer font name?

Comment (nehasri89) [reply .github/CONTRIBUTING.md]: this is a real fontname we are using in the times component

Comment (aronshamash) [reply .github/CONTRIBUTING.md]: and the `W04` part is needed and can't be removed ?

Comment (thatportugueseguy) [reply .github/CONTRIBUTING.md]: I just looked into the fonts on the fonts server and at the moment we're using the fonts without  the w04, but it's actually there on the metadata.

given that W04 is just a weight indication, maybe for simplicity we can ditch the W04??

https://www.thetimes.co.uk/fonts/TimesDigital-Regular ==>
screen shot 2017-10-30 at 13 55 56

**Comment by craigbilner** [comment]
@nehasri89 updated the wording
@aronshamash perhaps we keep the `W04` for now to be consistent and make it easier to replace as is?

**Comment by aronshamash** [comment]
yes lets leave `W04` so it is consistent.

**Pull request merged**